annotation/pom.xml

```diff
@@ -21,7 +21,7 @@
   <parent>
     <groupId>com.google.errorprone</groupId>
     <artifactId>error_prone_parent</artifactId>
-    <version>2.3.2-SNAPSHOT</version>
+    <version>2.2.1-SNAPSHOT</version>
```

Comment:
Hmm, I'm a bit puzzled by this drop in version. @awturner @ronshapiro Can you explain the reasoning behind this change for me?

Reply:
Hrm, it looks like when @eaftan ran maven-release-plugin, the changes weren't synced internally, so our source of truth is inconsistent. I'll fix this and it will be in the next change. Thanks for the close inspection!

Reply:
👍😊